src/cheevos/Cheevos.cpp

```diff
@@ -91,6 +103,48 @@ void CCheevos::EvaluateRichPresence(char* evaluation, size_t size)
   rc_evaluate_richpresence(m_richPresence, evaluation, size, PeekInternal, this, NULL);
 }
 
+void CCheevos::ActivateAchievement(unsigned cheevo_id, const char* memaddr)
+{
+  rc_runtime_activate_achievement(&m_runtime, cheevo_id, memaddr, NULL, 0);
+  // it will return integer value 0 in case achivement is activated successfully.
```

Comment:
`rc_runtime_activate_achievement` will return 0 if the achievement is activated successfully but the method `ActivateAchievement` returns `void`?

Reply:
Yes, I will make these changes after rebasing and adding new commits